src/hijackers/dynamicHeadAppend.ts

```diff
@@ -3,13 +3,32 @@
  * @since 2019-10-21
  */
 import { checkActivityFunctions } from 'single-spa';
+import { execScripts } from 'import-html-entry';
 import { Freer } from '../interfaces';
 
 const rawHtmlAppendChild = HTMLHeadElement.prototype.appendChild;
 
-export default function hijack(appName: string, bootstrapping = false): Freer {
+const SCRIPT_TAG_NAME = 'SCRIPT';
+const LINK_TAG_NAME = 'LINK';
+const STYLE_TAG_NAME = 'STYLE';
+
+export default function hijack(appName: string, proxy: Window, bootstrapping = false): Freer {
   let dynamicStyleSheets: HTMLLinkElement[] = [];
   HTMLHeadElement.prototype.appendChild = function appendChild<T extends Node>(this: any, newChild: T) {
+    if (!((newChild as any).tagName && [SCRIPT_TAG_NAME, LINK_TAG_NAME, STYLE_TAG_NAME].includes(newChild as any))) {
```

Comment:
> [SCRIPT_TAG_NAME, LINK_TAG_NAME, STYLE_TAG_NAME].includes(newChild as any))

should be [SCRIPT_TAG_NAME, LINK_TAG_NAME, STYLE_TAG_NAME].includes(newChild as any).tagName) ?

Reply:
sorry, I fix it.